src/com/qamp/kanita/zadaci/zadatak1/Swim.java

```diff
@@ -0,0 +1,7 @@
+package com.qamp.kanita.zadaci.zadatak1;
+
+public interface Swim {
+    String swim();
+
+    String getThing();
```

Comment:
I generally don't mind you (or anyone else) adding things that were not specified, but keep in mind that every method of an `interface` must be implemented. So this adds the additional burden of implementing a `getThing` to every class which implements either `Swim` of `Flight`.

Also, `interfaces` can be inherited. So since both Swim and Flight have the getThing method, you could have done something like:
```
public interface Thing {
    String getThing();
}

public interface Swim extends Thing {
    String swim();
}
```

This works exactly as what you did, anything implementing `Swim` must implement all methods of Swims parents.